client/components/deposits-overview/recent-deposits.tsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { Flex, FlexItem, Icon } from '@wordpress/components';
+import { calendar } from '@wordpress/icons';
+
+/**
+ * Internal dependencies.
+ */
+import strings from './strings';
+import './style.scss';
+import DepositStatusChip from 'components/deposit-status-chip';
+import { getDepositDate } from 'deposits/utils';
+import { CachedDeposit } from 'wcpay/types/deposits';
+import { useDeposits } from 'wcpay/data';
+import { formatCurrency } from 'wcpay/utils/currency';
+import { Link } from '@woocommerce/components';
+import { getDetailsURL } from 'wcpay/components/details-link';
+
+interface DepositRowProps {
+	deposit: CachedDeposit;
+}
+interface RecentDeposits {
+	deposits: CachedDeposit[];
+	isLoading: boolean;
+}
+interface RecentDepositsProps {
+	currency: string | undefined;
+}
+
+const tableClass = 'wcpay-deposits-overview__table';
+
+const GetRecentDeposits = ( currency?: string ): RecentDeposits => {
+	const query = {
+		status_is_not: 'estimated',
+		store_currency_is: currency,
+		orderby: 'date',
+		order: 'desc',
+		per_page: '3',
+	};
+	const deposits = useDeposits( query );
+
+	return {
+		deposits: deposits.deposits,
+		isLoading: deposits.isLoading,
+	};
+};
+
+const DepositTableRow: React.FC< DepositRowProps > = ( {
+	deposit,
+}: DepositRowProps ): JSX.Element => {
+	return (
+		<Flex className={ `${ tableClass }__row` }>
+			<FlexItem className={ `${ tableClass }__cell` }>
+				<Icon icon={ calendar } size={ 17 } />
+				<Link href={ getDetailsURL( deposit.id, 'deposits' ) }>
+					{ getDepositDate( deposit ) }
+				</Link>
+			</FlexItem>
+			<FlexItem className={ `${ tableClass }__cell` }>
+				<DepositStatusChip status={ deposit.status } isCompact />
```

Comment:
Another note that this will need to be updated to `<DepositStatusPill />` (added https://github.com/Automattic/woocommerce-payments/pull/5946). I don't think we should hold off merging this though.

We can update this inside that PR if this is merged before.